host/windows/posix.c

```diff
@@ -0,0 +1,580 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/*
+**==============================================================================
+**
+** windows/posix.c:
+**
+**     This file implements POSIX OCALLs for Windows. Most of these are stubs
+**     which are still under development.
+**
+**==============================================================================
+*/
+
+#include <io.h>
+#include <stdint.h>
+
+#include "posix_u.h"
+
+/* Definitions in <openenclave/corelibc/errno.h> with Windows headers. */
+#ifdef _OE_ERRNO_H
```

Comment:
I don't quite follow the errno implementation in this PR. errno is standard C. So it ought to be available in Windows too.
The EDL propagate_errno feature might simplify the errno handling and propagation.

Reply:
Thanks Anand. I removed that block of code since it is no longer necessary. I added it because the standard errno names conflict with similar names in Windows. I have since restored the OE_ prefixes in <openenclave/corelibc/errno.h> to resolve the issue.

Note that in this module we need to raise the POSIX errnos but still handle the Windows errnos (the two are overlapping).